### PR TITLE
RM - message options on all tabs

### PR DESCRIPTION
**Pull request by anvmn**
#605 

Per https://github.com/TIP-Global-Health/eheza-app/issues/605#issuecomment-1424357709

### CODE REVIEW

**Comment by anvmn** [comment]
Peek 2023-02-12 11-34

File: client/src/elm/Pages/MessagingCenter/View.elm [hunk 1]
```diff
@@ -506,18 +504,14 @@ viewResilienceMessage language nurseId nurse model ( messageId, message ) =
...
+            div [ class "title-wrapper" ]
+                [ plainTitle
+                , div
+                    [ class "options"
+                    , onClick <| SetMessageOptionsDialogState <| Just <| MessageOptionsStateMain ( messageId, message )
                     ]
-
-            else
-                plainTitle
+                    [ text "OP" ]
```
Comment (mariano-dagostino) [inline client/src/elm/Pages/MessagingCenter/View.elm]: @anvmn Do you plan to translate this text in the future?

**Review by mariano-dagostino — APPROVED** [review]
@anvmn Looks good. One small question.

File: client/src/elm/Pages/MessagingCenter/View.elm (hunk 1 above)
Comment (anvmn) [reply client/src/elm/Pages/MessagingCenter/View.elm]: @mariano-dagostino This will be replaced by icon

**Pull request merged**